scripts/goldpolish-target-extract-seq.py

```diff
@@ -0,0 +1,178 @@
+"""Extracts regions to polish with additional flanks into fasta file"""
+# imports
+import argparse
+import csv
+import re
+import btllib
+from collections import namedtuple
+
+Coordinate = namedtuple("Coordinate", "start end")
+min_gap_length = 1
```

Comment:
```suggestion
MIN_GAP_LENGTH = 1
```
Just good practice to use upper-case for constants